includes/admin.form.inc

```diff
@@ -126,6 +134,28 @@ function islandora_compound_object_admin_form_jail_validation(array $element, ar
 }
 
 /**
+ * Warn if both redirect to first and compound parent in breadcrumb are enabled.
+ *
+ * @param array $element
+ *   The element to check.
+ * @param array $form_state
+ *   The Drupal form state.
+ * @param array $form
+ *   The Drupal form definition.
+ */
+function islandora_compound_object_admin_form_redirect_to_first_validation(array $element, array &$form_state, array $form) {
+  if ($element['#value'] == 1) {
+    if (isset($form_state['values']['islandora_compound_object_show_compound_parents_in_breadcrumbs'])
+        && $form_state['values']['islandora_compound_object_show_compound_parents_in_breadcrumbs'] == 1) {
+      $first_title = rtrim($element['#title'], '.!?');
+      $parent_title = rtrim($form['islandora_compound_object_show_compound_parents_in_breadcrumbs']['#title'], '.!?');
+      drupal_set_message(t("Enabling both %first and %parent is not recommended, because the compound object parent in the breadcrumb will be redirect to the first child, so cannot be actually reached.",
```

Comment:
Some advice here @whikloj @jonathangreen (as native English representatives) 
I feel we need to be should use just one form. The compound object, the compound object parent, or any other. So if using compound object's parent here, we should maybe is it also at https://github.com/Islandora/islandora_solution_pack_compound/pull/130/files#diff-0cdd19abe14e51fe9e7f061ec72c86d3R100
Also, i still don't see a harm on "Display compound object parents", this should be just an advice type (Warning as defined now is ok  maybe?) . Mostly because even without actual parent access/ metadata, the breadcrumb shows the actual title of the parent, which in many cases defines semantic context!

Also, i really would like to see a warning/as this one if
```PHP
$form['islandora_compound_object_compound_children'] = array(
     '#type' => 'checkbox',
     '#title' => t('Only allow compound objects to have child objects associated with them'),
     '#description' => t('If unchecked, all objects may have child objects.'),
     '#default_value' => variable_get('islandora_compound_object_compound_children', TRUE),
   );
```` is enabled and redirect also. Because even if the functionality will be close, it will not be the same. No redirect on those objects, different managment of the parent, different behaviour of breadcrumb. Could that be done?
@whikloj @jonathangreen could you give us a hand here with the wording?

Reply:
@DiegoPino I haven't had a chance to try this out again, so I'll need to find time to do that before I can say.

As for the second concern, I still need to test but I think that if you use a large_image as the parent of your compound, then you get that object first. Including all metadata. Which is the same as you get by using a compoundCModel object with a large_image as the first child with this new toggle enabled.

Managing the parent is the same for both and the redirect states `Redirect to first child when a compound object is requested`. Which is not the same as requesting a large_image object with children 😉

Reply:
@whikloj thanks a lot. When you have some time please give my second concern a test. At least the management part (where to click) is actually not the same for the parent. But you are right, if I'm alert (morning coffee) and actually read that sentence calmly it is clear that it won't redirect if the parent is a newspaper. 🎉  Could be even 2xClearer if it says `"Redirect to the first child when an object of type compoundCMODEL is requested." or similar in actual good English. Actually, you are right. That piece should be ok. Still give it a spin when you can. Thanks